@types/modules/pm/groups.d.ts

```diff
@@ -0,0 +1,20 @@
+declare interface PMGroupUpdateRequest {
+    prizeCategoryId: string;
+    prizeCatagoryName?: string;
+    prizeCategoryDescription?: string;
+    prizeCategoryOrder?: Array<string>;
+  }
+
+  declare interface PMGroupCreateRequest extends PMGroupUpdateRequest {
+    prizeCatagoryName: string;
```

Comment:
In terms of attribute naming, can we go with "group" instead of "category", just so it's not confusing as to what objects are being edited? Also, if we don't declare id above, let's add it here.
```suggestion
    prizeGroupId: string;
    prizeGroupName: string;
```